File: debug_scripts/patch_font.py
import re

with open('C:\\Users\\MSI\\Desktop\\AI\\Odo\\index.html', 'r', encoding='utf8') as f:
    content = f.read()

# Replace all occurrences of font-family:monospace with font-family:'Calibri', sans-serif
content = content.replace("font-family:monospace", "font-family:'Calibri', sans-serif")
# Also handle cases with spaces
content = content.replace("font-family: monospace", "font-family: 'Calibri', sans-serif")

with open('C:\\Users\\MSI\\Desktop\\AI\\Odo\\index.html', 'w', encoding='utf8') as f:
    f.write(content)
print("Font patched!")

File: debug_scripts/patch_font.js
const fs = require('fs');
let code = fs.readFileSync('C:/Users/MSI/Desktop/AI/Odo/index.html', 'utf8');

const targetStr = `'<td style="font-family:Calibri, sans-serif; font-weight:600; color: var(--text-primary);">' + escapeHtml(data.bienSo) + '</td>\\n' +`;
const replacementStr = `'<td style="font-family:Calibri, sans-serif; font-weight:700; font-size:15px; letter-spacing: 0.8px; color: var(--text-primary);">' + escapeHtml(data.bienSo) + '</td>\\n' +`;

if (code.includes(targetStr)) {
    code = code.replace(targetStr, replacementStr);
    fs.writeFileSync('C:/Users/MSI/Desktop/AI/Odo/index.html', code);
    console.log('Replaced successfully');
} else {
    console.log('Target string not found');
}
